client/src/components/messages/Conversations.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import SendMessage from "./SendMessage";
+
+const Conversations = ({ message }) => {
+  return (
+    <div className="overflow-x-hidden w-full relative">
```

Comment:
What do you think of making the `text-primary` to make it take on the dark blue color? Do you prefer the text in these sections black? I'm not sure if the blue works better or not, it's consistent throughout the site, but what do you think here?

Reply:
Yeah they should use the text-primary property. I must have omitted that